projects/manager/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": ["projects/manager/tsconfig.app.json", "projects/manager/tsconfig.spec.json"],
+                "createDefaultProgram": true
+            },
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
+    ]
+}
```

Comment:
logout should be somewhat separated to the other menu items

Reply:
45cfc5e425278f4e7ee0131b0133d6db5e699ad3